lib/workers/repository/dependency-dashboard.ts

```diff
@@ -312,7 +314,8 @@ export async function ensureDependencyDashboard(
   }
 
   if (config.dependencyDashboardFooter?.length) {
-    issueBody += `---\n${config.dependencyDashboardFooter}\n`;
+    issueBody +=
+      `---\n` + template.compile(config.dependencyDashboardFooter, config);
```

Comment:
I don't see a need for backticks here
```suggestion
      '---\n' + template.compile(config.dependencyDashboardFooter, config);
```
or you should use them
```suggestion
      `---\n${template.compile(config.dependencyDashboardFooter, config)}`;
```